notebooks/125-torchvision-zoo-to-openvino/125-lraspp-segmentation.ipynb

```diff
@@ -0,0 +1,438 @@
+{
```

Comment:
<p>probably we need some consistency between 2 tutorials. </p><p>in previous one you use pretrained = True for weights loading, here you download weights manually using URL instead of just passing it into the model.</p><p> </p><p>Manual download was used in some old tutorials where we demonstrate general flow to describe situation when user has own pretrained model instead of model zoo. But for tutorials that teach how to use public models from zoos, it is better to utilize capability of zoo.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1345/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
changed